### DISCUSSION TITLE
Folder Restrictions on Default Folder Selector

### DISCUSSION THREAD

**Discussion by TRRESTGHGYTCFTGBCXSZD** [rafaelvcaetano/melonDS-android#1298] [Support]
For some reason, whatever i try to select the Download folder, it says "To protect your privacy, choose another folder", can't select on One UI 6.0, starting from android 11. It appears that certain folders are given an restriction and the only way to get around it is using custom folder selector after android 11, which does not have the restrictions.

**Comment by rafaelvcaetano** [comment]
As WinUser said, this is entirely controlled by Android. There is nothing I can do about that.

**Comment by yellow-flash02** [comment]
> For some reason, whatever i try to select the Download folder, it says "To protect your privacy, choose another folder", can't select on One UI 6.0, starting from android 11. It appears that certain folders are given an restriction and the only way to get around it is using custom folder selector after android 11, which does not have the restrictions.

 u can use folders within download folder not the default download folder so just create a new folder in download and paste ur roms there
